features/step_definitions/user_steps.rb

```diff
@@ -42,39 +42,44 @@
 Given(/^I am logged into the site$/) do
   visit new_user_session_path
   FactoryGirl.create(:user)
-  fill_in 'Email', with: 'newuser@sb.com' 
-  fill_in 'Password', with: 'password' 
+  fill_in 'Email', with: 'newuser@sb.com'
```

Comment:
@jahammo2: I can't do Faker for login, can I? I'm using my factory here instead since that user will exist. I did use Faker in my seeds file though and in all my unit tests.

Let me know if I'm missing something!